code/test/sixsq/nuvla/server/util/time_test.clj

```diff
@@ -0,0 +1,7 @@
+(ns sixsq.nuvla.server.util.time-test
+  (:require
+    [clojure.test :refer [deftest is]]
+    [sixsq.nuvla.server.util.time :refer [date-from-str]]))
+
+(deftest date-from-str-utc
+  (is (not (nil? (date-from-str "1964-08-25T10:00:00.00Z")))))
```

Comment:
function call to  not and nil? can be replaced with some?.

I think it's good to add more tests perhaps with different dates with millisecond without with time offset instead of Z.